neutral_generation/constants.py

```diff
@@ -1,10 +1,13 @@
 # pronouns
-NON_FUNCTION_PRONOUNS = {   # need to change pronoun based on context
+NON_FUNCTION_PRONOUNS = {  # need to change pronoun based on context
     # one-to-many mapping
-    'her': ['them',     # I talked to her --> I talked to them
-            'their'],   # This is her pen --> This is their pen
-    'his': ['their',    # This is his pen --> This is their pen
-            'theirs']   # This pen is his --> This pen is theirs
+    # using dependency parser for "her" --> "their" / "them", so not including "her" here
+    'his': ['their',  # This is his pen --> This is their pen
+            'theirs'],  # This pen is his --> This pen is theirs
+    "he's": ["they've",
+             "they're"],
+    "she's": ["they've",
```

Comment:
Add a comment for why you include "he's" and "she's" since they are handled differently -- i.e. you would expect to operate on the individual tokens "(s)he" and " 's".

Reply:
Good suggestion. Will do.